traffic_portal/app/src/common/modules/form/cdn/FormCDNController.js

```diff
@@ -17,7 +17,7 @@
  * under the License.
  */
 
-var FormCDNController = function(cdn, $scope, $location, $uibModal, formUtils, stringUtils, locationUtils, cdnService) {
+var FormCDNController = function(cdn, $scope, $location, $state, $uibModal, formUtils, stringUtils, locationUtils, cdnService, messageModel) {
```

Comment:
The only changes I see to this file are adding these  dependency injections and removing a blank line. Does this controller use those injections already, and it was a bug that they weren't being injected?

Reply:
i'll check